docs/Monorepos-Overview.md

```diff
@@ -0,0 +1,97 @@
+
+- **Small to medium-sized teams**: Monorepos work well for small to medium-sized teams. With everyone working in the same repository, it’s easier to collaborate, share code, and stay aligned on changes.
+
+## When not to use a monorepo
+
+- **Large and unrelated projects**: If your projects are large and unrelated, a monorepo can become unwieldy. The repository size might grow excessively, and it can become difficult to manage unrelated codebases together, leading to longer CI/CD times and more complex build processes.
+
+- **Different lifecycles**: When different projects have significantly different release cycles or stability requirements, maintaining them in a monorepo can introduce unnecessary complexity. For example, if one service is in active development while another is in maintenance mode, separate repositories might be more appropriate.
+
+- **Security concerns**: If your projects require different access controls or have varying levels of security requirements, a monorepo might not be suitable. Ensuring that sensitive parts of the codebase are protected can be more challenging in a monorepo setup.
+
+- **Large, distributed teams**: For large, distributed teams, a monorepo can become hard to manage. With many contributors, the repository can grow quickly, leading to longer build times and more complicated merges. In these cases, using separate repositories might be better, allowing different teams to work independently without slowing each other down.
+
+## Common monorepo tools
+
+| Tool                          | Pro                                                                                                                                       | Cons                                                                                                                             |
+| ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| Yarn Workspaces [Recommended] | Built-in support for monorepos. <br>Simplifies dependency management across packages. <br>Excellent community support and documentation.  | Can be complex to configure initially. <br>Some issues with peer dependencies.                                                   |
+| Lerna                         | Great for managing multi-package repositories. <br>Supports independent versioning of packages. <br>Integrates well with Yarn Workspaces. | Slower for large codebases. <br>Requires additional configuration for CI/CD.                                                     |
```

Comment:
Lerna is no longer actively supported. I would either not mention it or say that it's obsolete / not recommended.